.gitignore

```diff
@@ -94,3 +94,7 @@ db.sqlite3
 
 */migrations/*
 !*/migrations/__init__.py
+.vscode/
+.DS_Store
+authors/.DS_Store
```

Comment:
I think you can ignore all .DS_Store files in one line

Reply:
Noted, working on it